### DISCUSSION TITLE
Not really sure how to use the software.

### DISCUSSION THREAD

**Discussion by markstewart5** [synthetichealth/synthea#1117] [Q&A]
Hi, 
I am currently trying to generate some synthetic data for a process, i have created in the modules section. 
However, i do not know how to do this, and i cannot seem to even generate a synthetic population.  If anyone could provide any help on this matter that would be really appreciated. 
I have tried looking at youtube but all the videos assume prior use of java.

**Comment by jawalonoski** [comment]
Sorry, we can't provide getting started help with Java itself. There are a lot of good resources and videos out there dedicated on how to install the Java Virtual Machine or Java Development Kit. Our development team is currently testing with Java 8 and Java 17.

Once you have Java installed, you should be able to follow our getting started instructions https://github.com/synthetichealth/synthea/wiki/Basic-Setup-and-Running.

If you want to use a custom disease module, perhaps that you created with the [Module Builder](https://synthetichealth.github.io/module-builder/), you can use the `-d` command-line parameter to specify the folder where your module(s) exists.

For example, using the latest build that you download:

```
java -jar synthea-with-dependencies.jar -d my_modules_folder
```
or, if you are following the developer instructions:
```
./run_synthea -d my_modules_folder
```

Alternatively, you can also download some pre-generated data sets from https://synthea.mitre.org/downloads
